vllm/entrypoints/harmony_utils.py

```diff
@@ -29,6 +31,16 @@
 
 _harmony_encoding = None
 
+BUILTIN_TOOLS = {
+    "web_search_preview",
```

Comment:
add some comment about to explain the tools? What's input and output expectation?